src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/repository/StandardAddressCheckRequestRepository.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.StandardAddressCheckRequest
+
+interface StandardAddressCheckRequestRepository : JpaRepository<StandardAddressCheckRequest, Long>
```

Comment:
you can share the same repository for both and return different sub classes.. but maybe it makes sense to keep them separate?

Reply:
Cool didn't think about that, I look at adding that in my next PR. It may make sense as I can't see them having too many methods but some might get duplicated